CHANGELOG.next.asciidoc

```diff
@@ -98,6 +98,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Load DLLs only from Windows system directory. {pull}13234[13234] {pull}13384[13384]
 - Fix mapping for kubernetes.labels and kubernetes.annotations in add_kubernetes_metadata. {issue}12638[12638] {pull}13226[13226]
 - Fix case insensitive regular expressions not working correctly. {pull}13250[13250]
+- Fix module-level fields in Kubernetes metricsets. {pull}13433[13433]
```

Comment:
Should this be under a bugfix in Metricbeat session?

Reply:
Yes :man_facepalming: good catch

Reply:
Fixed here and in backports for #13433